_includes/footer.html

```diff
@@ -2,10 +2,10 @@
   <div class="footer-section-bottom bg-base-lighter">
     <div class="grid-container">
       <div class="grid-row padding-3">
-        <div class="tablet:grid-col" >
+        <div class="tablet:grid-col">
           <div class="logo-links">
             <a class="footer-logo media_link" href="https://18f.gsa.gov">
-              <img src="{{ site.baseurl }}/assets/uswds/img/logo-img.png" width="50" alt="Ageny logo" />
+              {% asset logo-img.png width="50" alt="Agency logo" %}
```

Comment:
I won't highlight all of these, but this is the `jekyll-assets` provided tag for including assets. (and fixed a typo!)